testsuite/openmodelica/interactive-API/ConversionVersions.mos

```diff
@@ -0,0 +1,28 @@
+// status: correct
+
+loadString("package P1
+  annotation(conversion(from(version={x,y,z})));
+end P1;");
+getConversionsFromVersions(P1);getErrorString();
+loadString("package P2
+  annotation(conversion(xfrom()));
+end P2;");
+getConversionsFromVersions(P2);getErrorString();
+loadString("package P3
+  annotation(conversion(noneFromVersion=\"1.2.3\",from(version=\"1.2.4\"),from(version={\"1.2.5\",\"1.2.6\"}),noneFromVersion=\"1.2.7\"));
+end P3;");
+getConversionsFromVersions(P3);getErrorString();
+
+// Result:
+// true
+// ({},{})
+// "[<interactive>:2:25-2:46:writable] Warning: Conversion-annotation is missing version for from-conversion: from(version = {x, y, z}).
```

Comment:
A bit late but just want to know why is this warning when `P1` does have `conversion(from(version={x,y,z}))`

Reply:
They are not strings

Reply:
Yes that's what I expected so the warning is misleading. Better warning would be to tell the user that annotation format is wrong instead of saying missing.